src/Internal.AspNetCore.HostingStartup.Sdk/buildMultiTargeting/Internal.AspNetCore.HostingStartup.Sdk.props

```diff
@@ -0,0 +1,7 @@
+<!--
+  This file is automatically imported by NuGet into a user's project
```

Comment:
What are the scenarios for mutli-targeting hosting startup projects? Adding these imports is unlikely to work in a multi-target scenario since these projects invoke a new project build context per TFM.

Reply:
I just copied this from the sample. Would it work fine if I remove them?

Reply:
Yup, it should work ok